rotkehlchen/chain/ethereum/modules/uniswap/uniswap.py

```diff
@@ -594,7 +594,7 @@ def _get_events_graph(
                     address=address,
                     timestamp=Timestamp(int(event['timestamp'])),
                     event_type=event_type,
-                    pool_address=to_checksum_address(event['pair']['id']),
+                    pool_address=deserialize_ethereum_address(event['pair']['id']),
```

Comment:
same thing here

Note: All the ones that are in the same function/for loop it would make sense to have them deserialized together in a `try/except`